src/components/ProjectContributors.js

```diff
@@ -211,6 +213,7 @@ const ProjectContributors = (props) => {
             <h1>
                 {`Add new contributors to the project`}
             </h1>
+            { errorGithubContributors ? isNotAContributor() : null }
```

Comment:
Remove space after opening brace and before closing brace

`{errorGithubContributors ? isNotAContributor() : null}`

or

```
{errorGithubContributors
    ? isNotAContributor()
    : null
}
```